moped-editor/src/queries/project.js

```diff
@@ -546,39 +546,6 @@ export const PROJECT_ARCHIVE = gql`
   }
 `;
 
-export const PROJECT_CLEAR_MAP_DATA_TEMPLATE = `mutation ClearProjectMapData($projectId: Int!) {
```

Comment:
deleted this because I couldnt find it being used anywhere, but if it should stay let me know and I'll put it back.